### PR TITLE
Feat/wrap additional properties

### PR DESCRIPTION
**Pull request by jacqueswho**
### Reasons for making this change

fixes [1927](https://github.com/rjsf-team/react-jsonschema-form/issues/1927#issuecomment-1041197937)

### Checklist

* [ ] **I'm updating documentation**
  - [ ] I've [checked the rendering](https://react-jsonschema-form.readthedocs.io/en/latest/#contributing) of the Markdown text I've added
* [x] **I'm adding or updating code**
  - [x] I've added and/or updated tests
  - [ ] I've updated [docs](https://react-jsonschema-form.readthedocs.io/) if needed
  - [ ] I've updated the [changelog](https://github.com/rjsf-team/react-jsonschema-form/blob/master/CHANGELOG.md) with a description of the PR
* [x] **I'm adding a new feature**
  - [ ] I've updated the playground with an example use of the feature

### CODE REVIEW

File: packages/core/src/components/Form.js [hunk 1]
```diff
@@ -462,6 +462,7 @@ export default class Form extends Component {
     const { schema, uiSchema, formData, errorSchema, idSchema } = this.state;
     const registry = this.getRegistry();
     const _SchemaField = registry.fields.SchemaField;
+
```
Comment (heath-freenome) [inline packages/core/src/components/Form.js:465]: Probably can revert this if you want

File: packages/semantic-ui/src/FieldTemplate/FieldTemplate.js [hunk 2]
```diff
@@ -15,28 +16,43 @@ function FieldTemplate({
...
-        <RawErrors errors={rawErrors} options={errorOptions} />
-      </MaybeWrap>
-    </Form.Group>
-  );
+  
+  if (hidden) {
+    return children;
+  }
+
+  return (<WrapIfAdditional
```
Comment (heath-freenome) [inline packages/semantic-ui/src/FieldTemplate/FieldTemplate.js:31]: Did you mean to change the pattern from:
```
return (
 <xxx>
   <yyy />
 </xxx>
);
```
to
```
return (<xxx>
  <yyy/>
</xxx>);
```
???

Comment (jacqueswho) [reply packages/semantic-ui/src/FieldTemplate/FieldTemplate.js:31]: not sure, will remove, saw it on antd implementation.

**Review by heath-freenome — APPROVED** [review]
Just the one suggestion and otherwise, looks good

**Pull request merged**